chef/cookbooks/resolver/recipes/default.rb

```diff
@@ -51,6 +51,9 @@
     end
     not_if { node["crowbar"]["admin_node"] && File.exist?("/var/lib/crowbar/install/disable_dns") }
   end
+  utils_systemd_service_restart "dnsmasq" do
+    not_if { node["crowbar"]["admin_node"] && File.exist?("/var/lib/crowbar/install/disable_dns") }
```

Comment:
Why is this line necessary?  Wouldn't setting the Restart option be irrelevant if the service is disabled?  Makes me wonder if the `not_if` in the previous `service` block is actually the wrong approach because if someone had manually enabled it, it wouldn't disable it.

Reply:
You're correct, it's irrelevant, but on the other hand, it makes little sense to create a snippet if we don't do anything for the service. So somehow, it doesn't hurt either, and I view it more as consistency between the two resources.